contracts/interfaces/IPoolConfigurator.sol

```diff
@@ -310,6 +317,14 @@ interface IPoolConfigurator {
    **/
   function setReserveStableRateBorrowing(address asset, bool enabled) external;
 
+  /**
+   * @notice Configures flashloans on a reserve
+   * @dev Can only be enabled (set to true) if borrowing is enabled
```

Comment:
This is not true. Only needed if the flashloan turns into a borrow.

Reply:
True - I initially implemented this to be true and realized this was not a required condition. Updating the comment.